doc/dev/development.rst

```diff
@@ -290,8 +329,15 @@ A typical PypeIt development workflow is as follows:
 
    .. code-block:: bash
 
-        cd $PYPEIT_DIR/pypeit/tests
-        pytest .
+        cd $PYPEIT_DIR
+        pytest
+
+   or preferably:
+
+   .. code-block:: bash
+
+        cd $PYPEIT_DIR
+        tox -e test
```

Comment:
i take it you have `PYPEIT_DEV` set, correct? because i think if you don't, that test gets skipped. so try doing `tox -e test-pyside2-shapely` so that you have the same env as the one `pytest` is using.

Reply:
Nice.  That fixed it.  Thanks!